udp-echo/Cargo.toml

```diff
@@ -5,7 +5,9 @@ authors = ["Anton Patrushev <apatrushev@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-actix = "0.7"
-tokio = "0.1"
-futures = "0.1"
-bytes = "0.4"
+actix = "0.9"
+tokio = "0.2"
+tokio-util = { version = "0.3", features = [ "udp" ] }
```

Comment:
`¨codec¨` feature seams to be missing here

Reply:
Interesting. For me it compiles even if I don't enable codec, even though according the docs it is needed for everything in `tokio_util::codec::*`. I added it.